### PR TITLE
change the traking update to weekly

### PR DESCRIPTION
**Pull request by Jin-Sun-tts**
# Pull Request

Related to [change tracking update from nightly job to weekly#4345](https://github.com/GSA/data.gov/issues/4345)

## About

change tracking update cron job from nightly job to weekly

### CODE REVIEW

**Review by FuhuXia — CHANGES_REQUESTED** [review]
Changes to another line is also needed.

`# Make sure 'schedule-cron' matches these varaibles.`

File: .github/workflows/ckan_auto.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: 4 - Automated CKAN Jobs
 
 on:
   schedule:
-    - cron: '30 7 * * *'       # Tracking Update -- every day at 2:30am EST
+    - cron: '30 7 * * 5'       # Tracking Update -- every Friday at 2:30am EST
```
Comment (FuhuXia) [inline .github/workflows/ckan_auto.yml:6]:
```suggestion
    - cron: '30 7 * * 0'       # Tracking Update -- every Sunday at 2:30am EST
```

File: .github/workflows/ckan_auto.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ on:
 env:
   ERROR: false
   # Make sure 'schedule-cron' matches these varaibles.
-  SCHEDULE_TRACKING: '30 7 * * *'
+  SCHEDULE_TRACKING: '30 7 * * 5'
```
Comment (FuhuXia) [inline .github/workflows/ckan_auto.yml:16]:
```suggestion
  SCHEDULE_TRACKING: '30 7 * * 0'
```

**Review by FuhuXia — CHANGES_REQUESTED** [review]
Let us do sunday not friday.

File: .github/workflows/ckan_auto.yml (hunk 1 above)
Comment (Jin-Sun-tts) [reply .github/workflows/ckan_auto.yml:6]: sure, no problem.

File: .github/workflows/ckan_auto.yml (hunk 2 above)
Comment (Jin-Sun-tts) [reply .github/workflows/ckan_auto.yml:16]: sure, no problem.

**Review by FuhuXia — APPROVED** [review]

**Pull request merged**